modules/ROOT/pages/index.adoc

```diff
@@ -1,6 +1,18 @@
-= An Example Quick Course
+= Welcome
 :navtitle: Home
 
-== Introduction
+This quick course covers the foundations of Red{nbsp}Hat OpenShift Data Science (RHODS).
 
-This is an example quick course demonstrating the usage of Antora for authoring and publishing quick courses.
\ No newline at end of file
+== Prerequisites
```

Comment:
Need more sub-sections on this page that talks about the classroom environment and how to provision (not detailed steps but we need to tell students which catalog item from RHD to order